doc/002-local-dev.md

```diff
@@ -7,56 +7,40 @@ This local development README guides you to install the Ziti stack built from th
 You will need to [install a version of Go](https://go.dev/) that is as recent as the version used by this project. Find the current minimum version by running this command to inspect `go.mod`.
 
 ```bash
-grep -Po '^go\s+\K\d+\.\d+$' go.mod
+grep -Po '^go\s+\K\d+\.\d+(\.\d+)?$' go.mod
```

Comment:
This improves the Perl regexp that matches the semver in our go.mod file so that it's more resilient to change. Without this improvement, the regexp will fail to match if we start expressing the Go version as a patch-level semver. This allows the patch level to be optional. Go.dev's latest release is 1.20, for example, without the patch level of the semver. They appear to alternate between minor level and patch level in their download URL patterns and associated download directory structure, hence the need for flexibility at every level when refering to the Go release version.